src/test/kotlin/lotto/domain/LottoMachineTest.kt

```diff
@@ -45,21 +34,66 @@ class LottoMachineTest {
     )
     fun `로또를 판매 시 받는 돈이 로또금액에 맞게 떨어지 않는 경우 예외를 반환한다`(lottoPrice: Int, money: Int) {
         val expectedMessage = "로또 구매 후 남은 돈이 있을 수 없습니다. money: $money, lottoPrice: $lottoPrice"
-        val lottoMachine = LottoMachine(lottoPrice = lottoPrice, lottoGenerator = FakeLottoGenerator())
-        val result = assertThrows<java.lang.IllegalArgumentException> { lottoMachine.sellLottos(money = money) }
+        val lottoMachine = LottoMachine(lottoPrice = Money(lottoPrice), lottoGenerator = FakeLottoGenerator())
+        val result = assertThrows<java.lang.IllegalArgumentException> { lottoMachine.sellLottos(money = Money(money)) }
```

Comment:
코틀린의 `IllegalArgumentException`을 사용하면 어떨까요?

```suggestion
        val result = assertThrows<IllegalArgumentException> { lottoMachine.sellLottos(money = Money(money)) }
```

Reply:
<img width="552" alt="CleanShot 2021-03-28 at 12 13 40@2x" src="https://user-images.githubusercontent.com/25674959/112741249-0af0a280-8fbf-11eb-9c26-3eec20a38942.png">

suggest에서 `자바 예외` 가 `코틀린 예외` 보다 먼저 나와서 실수를 자주 하게되네요. 😭  ae18566